library/core/resource_loader/src/testmod/java/org/quiltmc/qsl/resource/loader/test/StaticResourcesTestMod.java

```diff
@@ -0,0 +1,60 @@
+package org.quiltmc.qsl.resource.loader.test;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+import net.minecraft.block.AbstractBlock;
+import net.minecraft.block.Block;
+import net.minecraft.block.Blocks;
+import net.minecraft.registry.Registries;
+import net.minecraft.registry.Registry;
+import net.minecraft.resource.MultiPackResourceManager;
+import net.minecraft.resource.Resource;
+import net.minecraft.resource.ResourceType;
+import net.minecraft.util.Identifier;
+import org.quiltmc.loader.api.ModContainer;
+import org.quiltmc.qsl.base.api.entrypoint.ModInitializer;
+import org.quiltmc.qsl.resource.loader.api.ResourceLoader;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Predicate;
+
+public class StaticResourcesTestMod implements ModInitializer {
+	public static final Logger LOGGER = LoggerFactory.getLogger("QuiltStaticResourcesTest");
+	@Override
+	public void onInitialize(ModContainer mod) {
+
+		try {
+			MultiPackResourceManager clientManager = ResourceLoader.getStaticResourceManager(ResourceType.CLIENT_RESOURCES);
+			Resource cronch = clientManager.getResource(new Identifier("cronch", "test_client")).get();
+			BufferedReader readerCronch = cronch.openBufferedReader();
+			LOGGER.error(readerCronch.readLine() + "(Reading this line should be impossible!)");
```

Comment:
Please use the var-args system of the logger to avoid passing unsafe strings.
Given this is a test mod it's not as bad, but they're also references on how to use an API, so they should not accidentally give bad practices.

Reply:
Fixed in next commit.